lib/icingadb/icingadb-objects.cpp

```diff
@@ -996,6 +996,8 @@ void IcingaDB::InsertObjectDependencies(const ConfigObject::Ptr& object, const S
 					// Stringify if set.
 					if (values->Get(attr, &value)) {
 						switch (value.GetType()) {
+							case ValueEmpty:
+								break;
```

Comment:
To me break is redundant, but that's a matter of preference.

Reply:
Well, I would remove it as well.